tests/platforms/web/static/test_run_dev.py

```diff
@@ -1,25 +1,67 @@
-import re
+from unittest import mock
 
 import pytest
 
 from briefcase.exceptions import UnsupportedCommandError
+from briefcase.integrations.subprocess import Subprocess
+from briefcase.integrations.virtual_environment import VenvContext
 from briefcase.platforms.web.static import StaticWebDevCommand
 
 
 @pytest.fixture
 def dev_command(dummy_console, tmp_path):
-    return StaticWebDevCommand(
+    command = StaticWebDevCommand(
         console=dummy_console,
         base_path=tmp_path / "base_path",
         data_path=tmp_path / "briefcase",
     )
+    command.tools.home_path = tmp_path / "home"
+    command.tools.subprocess = mock.MagicMock(spec_set=Subprocess)
+    command._stream_app_logs = mock.MagicMock()
+    return command
 
 
-def test_run_dev_app_unsupported(dev_command, first_app_built):
-    with pytest.raises(
-        UnsupportedCommandError,
-        match=re.escape(
-            "The dev command for the web static format has not been implemented (yet!)."
-        ),
-    ):
-        dev_command.run_dev_app(first_app_built, env={})
+@pytest.fixture
+def default_venv() -> VenvContext:
+    """Create a venv mock for tests that require a venv parameter."""
+    mock_venv = mock.MagicMock(spec=VenvContext)
+    mock_venv.run.return_value = mock.MagicMock()
+    mock_venv.check_output.return_value = ""
+    mock_venv.Popen.return_value = mock.MagicMock()
+    mock_venv.executable = "/mock/venv/bin/python"
+    return mock_venv
+
+
+def test_run_dev_app_raises_unsupported_command_error(dev_command, first_app_config):
+    """Test that run_dev_app raises UnsupportedCommandError for web platform."""
+    with pytest.raises(UnsupportedCommandError) as excinfo:
+        dev_command.run_dev_app(
+            first_app_config,
+            env={},
+            venv=mock.MagicMock(spec=VenvContext),
+            passthrough=[],
+        )
+
+    assert excinfo.value.platform == "web"
+    assert excinfo.value.output_format == "static"
+    assert excinfo.value.command == "dev"
+
+
+def test_install_dev_requirements(dev_command, first_app_config, default_venv):
+    """Test that install_dev_requirements works with web dev command."""
+    # Mock the app with some requirements
+    first_app_config.requires = ["requests", "flask"]
+    first_app_config.test_requires = ["pytest"]
+
+    dev_command.install_dev_requirements(first_app_config, default_venv)
+
+    # Should call venv.run with pip install command
+    default_venv.run.assert_called_once()
+    call_args = default_venv.run.call_args[0][0]
+
+    # Check that it's calling pip install with the right packages
+    assert "pip" in call_args
+    assert "install" in call_args
+    assert "requests" in call_args
+    assert "flask" in call_args
+    assert "pytest" in call_args
```

Comment:
"requests flask pytest pip install" would satisfy this set of assertions. I thin you're looking for `assert_called_once_with(...)`